prime-router/src/main/kotlin/cli/ProcessFhirCommands.kt

```diff
@@ -106,26 +113,41 @@ class ProcessFhirCommands : CliktCommand(
      */
     private fun convertToHl7(jsonString: String): Message {
         return when {
-            fhirToHl7Schema == null ->
+            transformSchema == null ->
                 throw CliktError("You must specify a schema.")
 
-            !fhirToHl7Schema!!.canRead() ->
-                throw CliktError("Unable to read schema file ${fhirToHl7Schema!!.absolutePath}.")
+            !transformSchema!!.canRead() ->
+                throw CliktError("Unable to read schema file ${transformSchema!!.absolutePath}.")
 
             else -> {
                 val bundle = FhirTranscoder.decode(jsonString)
                 FhirToHl7Converter(
-                    fhirToHl7Schema!!.name.split(".")[0], fhirToHl7Schema!!.parent,
-                    context = FhirToHl7Context(
-                        CustomFhirPathFunctions(),
-                        null,
-                        translationFunctions = CustomTranslationFunctions()
-                    )
+                    transformSchema!!.name.split(".")[0], transformSchema!!.parent,
```

Comment:
Why did we change this? This should stay.

Reply:
was a merging issue. Fixed now